python/taichi/ui/scene.py

```diff
@@ -301,14 +312,17 @@ def mesh_instance(self,
         if transforms and (transforms.m != 4 or transforms.n != 4):
             print("Error! Transform matrix must be 4x4 shape")
             exit()
+        if display_mode < 0 or display_mode >= 3:
+            print("Error! display_mode must be 0(Fill), 1(Line), 2(Point)")
+            exit()
```

Comment:
usually a better way to do checks is raising errors instead of explicit print + exit

Reply:
Sorry, I will fix that.

Reply:
Have changed to raise exception.